teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>Although overwriting variables is not encouraged, I think it is ok in this instances because you are slicing away columns (and we do not use ar_data anywhere again). So I suggest to rename <code>ar_data</code> with <code>ar_dataset</code> - and then overwrite the variable:</p><p> </p><pre class="ql-syntax" spellcheck="false">ar_dataset = ar_dataset[['SMILES', 'accession', 'pchembl_value_Mean']]
ar_dataset
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>